tests/README.rst

```diff
@@ -61,12 +64,15 @@ Executing the tests can be done with::
 
 To run a single test do::
 
-    make TESTS=nosetests.sh check
+    make TESTS=unit_tests.sh check
+
 
+To run a subset of unit tests do::
 
-To run a subset of nose tests do::
+    make TESTS=unit_tests.sh UNIT_TESTS_PATTERN='test_layout_variant_' check
 
-    make TESTS=nosetests.sh NOSE_TESTS_ARGS="nosetests/pyanaconda_tests/keyboard_test.py nosetests/pyanaconda_tests/timezone_test.py" check
+The ``UNIT_TESTS_PATTERN`` variable is passed to ``unittest discover -k``. See
+the documentation for more info.
```

Comment:
Might be a good idea to link to the respective documentation or mention what documentation to check - man page or some Python module documentation ?